packages/wallet-sdk/src/sign/walletlink/WalletLinkSigner.ts

```diff
@@ -298,10 +294,8 @@ export class WalletLinkSigner implements Signer {
     const params = request.params || [];
 
     switch (method) {
-      case 'eth_requestAccounts': {
-        await this.handshake();
-        return { jsonrpc: '2.0', id: 0, result: this._addresses };
-      }
+      case 'eth_requestAccounts':
+        return this._eth_requestAccounts();
```

Comment:
reverting https://github.com/coinbase/coinbase-wallet-sdk/pull/1343/files#diff-5349a68685ed4873857c8ed4217ffaecf2878bc47719a3adf5f3a1b043a9e31b
will revisit this with overall WL refactor. for now, let's keep consistent with other methods.